Azure/azure-cli
# [AKS] `az aks nodepool update`: Add `--enable/disable-fips-image` flags for GA mutable fips

**Related command**
az aks nodepool update --enable-fips-image --disable-fips-image

**Description**
This PR allows the CLI functionality for the GA AKS feature allowing fips mode to be enabled/disabled after nodepool creation using the cli "az aks nodepool update" with associated flags above.

**Testing Guide**
#! Make sure to use region where AKS RP functionality has been released
az aks nodepool add -g chrislopezrg --cluster-name chrislopezaks -n fipstest2
az aks nodepool update -g chrislopezrg --cluster-name chrislopezaks -n fipstest --enable-fips-image
az aks nodepool show -g chrislopezrg --cluster-name chrislopezaks -n fipstest

az aks nodepool add -g chrislopezrg --cluster-name chrislopezaks -n fipstest3 --enable-fips-image
az aks nodepool add -g chrislopezrg --cluster-name chrislopezaks -n fipstest3 --disable-fips-image
az aks nodepool show -g chrislopezrg --cluster-name chrislopezaks -n fipstest3

**History Notes**


[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.
